hr_timesheet_employee_analytic_tag/models/hr_employee.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    analytic_tag_ids = fields.Many2many(
```

Comment:
Preventively avoid collapsing with other modules adding analytic tags to other things:

```suggestion
    timesheet_analytic_tag_ids = fields.Many2many(
```